test/e2e/new-dns-record.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from '@playwright/test';
+
+import { loggedInAsUser } from './utils';
+
+test.describe('not authenticated', () => {
+  test('redirects to login page', async ({ page }) => {
+    await page.goto('/');
+    await expect(page).toHaveURL(/.*login.*/);
+  });
+});
+
+test.describe('authenticated as user', () => {
+  loggedInAsUser();
+
+  test('redirects to edit dns record page when required fields are filled', async ({ page }) => {
+    await page.goto('/domains/new');
+    await page.getByLabel('Record Name*').fill('test');
+    await page.getByRole('combobox', { name: 'Type' }).selectOption('A');
+    await page.getByLabel('Value*').fill('test');
```

Comment:
Please let me know if I am wrong, but adding an asterisk to the value being passed is the playwright-specific syntax to indicate the field is not optional.

Reply:
Oh, the label has an asterisk next to the required fields on the page. This test was autogenerated by using the VS extension for playwright. So, it grabs the input with that label (`Record Name*`) and fills in the value

Reply:
Thanks for clarifying.